aosp_diff/preliminary/packages/services/Telecomm/01_0001-Handle-null-bindings-returned-from-ConnectionService-.patch

```diff
@@ -0,0 +1,64 @@
+From e18fc07621dedb7ea59a4424e4ad9036108b6a15 Mon Sep 17 00:00:00 2001
+From: Tyler Gunn <tgunn@google.com>
+Date: Mon, 7 Mar 2022 09:32:42 -0800
+Subject: [PATCH] Handle null bindings returned from ConnectionService.
+
+When a ConnectionService returns a null binding, immediately unbind from
+the ConnectionService and cancel any ongoing calls related to it.
+
+Bug: 211114016
+Test: Added new CTS test to verify auto unbind from null binding ConnectionService.
+Test: Manually tested using test app which implements null binding ConnectionService and verified via telecom log inspection that the service is unbound and the call is terminated.
+Change-Id: I0757557e66725dddfd871cd9857071a8749bd7ba
+(cherry picked from commit 410ce026004bb485c39afcc7d86e89d26ff1af94)
+Merged-In: I0757557e66725dddfd871cd9857071a8749bd7ba
+---
```

Comment:
Please cross-check if this patch is already present as a part of previous tag. If yes please remove this patch and update the commit id details in the excel sheet you maintain.
I can see it as already applied in your EB stdio logs.
Project ./packages/providers/MediaProvider
                Already applied         04_0004-Avoid-apps-to-get-global-access-with-table-uri.patch